app/services/school_partnerships/create.rb

```diff
@@ -7,7 +7,7 @@ def initialize(school:, lead_provider_delivery_partnership:)
       @lead_provider_delivery_partnership = lead_provider_delivery_partnership
     end
 
-    def create
+    def call
```

Comment:
I think we wanted to move away from standard service names like `call` and name actions to what they represent, so maybe its ok to keep to create I think

Reply:
Thanks for your suggestion to overcome the rubocop error! I've dropped that commit and made the change to the test in [this commit](https://github.com/DFE-Digital/register-early-career-teachers-public/pull/1356/commits/06070fae3f4bbdc25571df8d462a2c4df4125217).